### DISCUSSION TITLE
Excel Based reports are not getting generated after version 4.10 of BIRT.

### DISCUSSION THREAD

**Discussion by poonam197shukla** [eclipse-birt/birt#1269] [Q&A]
Error : java.lang.NullPointerException: Cannot invoke "com.binaryfortis.data.xls.XlsHelper$Range.getRow(int)" because "this.range" is null.
Can anyone please suggest a way to handle this scenario.

**Comment by hvbtup** [comment]
Not a BIRT issue. Contact whoever is responsible for "binaryfortis".
